uber/site_sections/preregistration.py

```diff
@@ -258,6 +258,8 @@ def register_group_member(self, session, group_id, message='', **params):
 
                 badge_being_claimed = group.floating[0]
                 attendee.registered = badge_being_claimed.registered
```

Comment:
That's a good point; it probably would be better if unassigned group badges which are Staff/Supporter just have number 0, and then this function would assign them the next available number when they're claimed.  Something for next year, I guess.  (Care to make a github issue for this?  I'm at work and trying to minimize my time-spent-on-MAGFest stuff </excuse>)